frappe/commands/site.py

```diff
@@ -1251,6 +1254,10 @@ def trim_database(context, dry_run, format, no_backup):
 			if format == "text":
 				click.secho(f"No ghost tables found in {frappe.local.site}...Great!", fg="green")
 		else:
+			print("Following tables will be dropped:")
+			print("\n".join(TABLES_TO_DROP))
+			pause_exec(10)
```

Comment:
Add a non-interactive flag to the command too?

Reply:
You mean `--yes`? That's current behaviour :eyes: we dont ask for confirmation, just wait before executing so user can ctrl+c if they want. 

IMO it's worth breaking this and just adding `-y` as not confirming can be quite bad.